### DISCUSSION TITLE
Stretching the Mesh in Turbine Case

### DISCUSSION THREAD

**Discussion by Sushmitharao46** [xcompact3d/Incompact3d#172] [Q&A] [answered]
Hi,
Will the ALM simulations work fine with mesh stretched in y direction.
I'm trying to simulate a case with stretching in y-direction. But I'm getting undesirable results. I've found that the **Su, Sv, Sw** in **acl_source.f90** assigned assigned the value **0** for the blade stations when the blade reaches to a particular Azimuth Angle. I've attached the **Cp** variation over time for the case I'm running. Is it because of stretching the mesh or something else.
Thank you.
![TurbineCp](https://user-images.githubusercontent.com/111350044/232027321-7f284fb6-8ef9-4b7d-9e98-0c962ae34fab.jpg)

**Comment by slaizet** [comment, marked as answer]
Hi,
The wind turbine subroutines have not been designed (yet) to work with a stretched mesh.
Best,
Sylvain

**Reply by Sushmitharao46** [reply]
Ok. Thank you
